### ISSUE TITLE
А график то кривой

### ISSUE THREAD

**Issue by majesty-dump** [AngelloUI/numerical-methods#1]
На самом деле, строит Chart.js "все правильно", но в своем понимании. Просто значения x вы передаете как labels, а библиотека рендерит их в прямом смысле как лейблы. Ей совершенно плевать, как они располагаются в численном множестве: для нее эти значения x равны листу с названиями месяцов или дней - она расположит их равноудаленно == с "шагом 1" .Она считает, что точки все равно находятся на равном расстоянии, просто по x они называются так, как вы передаете через лист var x. Поэтому и получается такая кривая херня. Если бы вы передавали в качестве данных равно-удаленные точки, то и отрисовывало бы нормально. 
Тот же matplotlib понимает, что мы работаем в R множестве, поэтому x и y для него - в классическом понимании координаты:
![изображение](https://user-images.githubusercontent.com/39465503/205512767-43008cb2-9d09-46f0-a8c0-5f069982fe7a.png)

А вот у Chart.js по умолчанию такого понимания нет, поэтому у него есть шкала значений и "названия точек по второй оси" (ВНИМАНИЕ НА ОСЬ X):
![изображение](https://user-images.githubusercontent.com/39465503/205512877-505c3389-74df-49b6-8ed1-17ae7da8c6dc.png)

**Comment by majesty-dump** [comment]
Вывод : проверяй инструменты, которые используешь. ~~Да и вообще, пошел этот JS нах...~~

**Comment by AngelloUI** [comment]
Да, действительно. Но тогда выбор(chartjs) казался нормальным. Спасибо.
